lib/tpm2_options.c

```diff
@@ -256,6 +256,9 @@ static void show_version (const char *name) {
 void tpm2_print_usage(const char *command, struct tpm2_options *tool_opts) {
     unsigned int i;
 
+    if (!tool_opts || !tool_opts->show_usage)
+        return;
```

Comment:
brackets

Reply:
@williamcroberts sorry... I've been repeatedly getting wrong this. Lately did some work on a different project whose coding style has the "single statements not using brackets" rule so I get confused :)

Reply:
Yeah that'll happen, been there. Also switch between C and python a lot, that can be brutal sometimes.